api/Window.json

```
@@ -589,13 +590,15 @@
           "status": {
             "experimental": false,
             "standard_track": false,
-            "deprecated": true
+            "deprecated": false
           }
         }
       },
       "beforeinstallprompt_event": {
         "__compat": {
           "description": "<code>beforeinstallprompt</code> event",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/BeforeInstallPromptEvent",
```

This should actually be:

```suggestion
          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Window/beforeinstallprompt_event",
```

...but this page doesn't exist yet.  A page for `onbeforeinstallprompt` does, however -- so maybe we should link to that for now on the assumption that we'll be merging the two pages in the future?  (cc @Elchi3)

```suggestion
          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Window/onbeforeinstallprompt",
```

---

https://developer.mozilla.org/docs/Web/API/Window/beforeinstallprompt_event would be the correct link.

https://developer.mozilla.org/docs/Web/API/BeforeInstallPromptEvent is the interface and not the event. 

https://developer.mozilla.org/docs/Web/API/Window/onbeforeinstallprompt is what we're stopping to do.